stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal

```diff
@@ -35,58 +39,76 @@ public type ServiceInformation {
     SecurityRequirement[] security,
 };
 
-@Description {value: "Model for Swagger contact information"}
-@Field {value: "name: Contact name"}
-@Field {value: "email: Contact email"}
-@Field {value: "url: Contact web address/page"}
+documentation {
+    Model for Swagger contact information.
+
+    F{{name}} Contact name
+    F{{email}} Contact email
+    F{{url}} Contact web address/page
+}
 public type Contact {
     string name,
     string email,
     string url,
 };
 
-@Description {value: "Model for service licence information"}
-@Field {value: "name: License name"}
-@Field {value: "url: License url"}
+documentation {
+    Model for service licence information.
+
+    F{{name}} License name
+    F{{url}} License url
+}
 public type License {
     string name,
     string url,
 };
 
-@Description {value: "Model for service documentation definition"}
-@Field {value: "description: Documentation description"}
-@Field {value: "url: Documentation url"}
+documentation {
+    Model for service documentation definition.
+
+    F{{description}} Documentation description
+    F{{url}} External documentation url
+}
 public type DocumentationInformation {
     string description,
     string url,
 };
 
-@Description {value: "Model for swagger service tag definition"}
-@Field {value: "name: Tag name"}
-@Field {value: "description: Tag decription"}
-@Field {value: "externalDocs: Optional documentation on the tag"}
+documentation {
+    Model for swagger service tag definition.
+
+	F{{name}} Tag name
+	F{{description}} Tag decription
+	F{{externalDocs}} Optional documentation on the tag
+}
 public type Tag {
     string name,
     string description,
     DocumentationInformation externalDocs,
 };
 
-@Description {value: "Model for security requirement definition. This is most likely the oauth scopes"}
-@Field {value: "name: Security scheme name"}
-@Field {value: "requirements: Array of security requirements"}
+documentation {
+    Model for security requirement definition. This is most likely the oauth scopes.
+
+	F{{name}} Security scheme name
```

Comment:
Seems like there are some inconsistancies when documenting fields, etc. Some are aligned with with the description and some are not. Shall we align with description in all places?

Reply:
@Shan1024 Checked locally. Code is properly formatted. Should be some issue in github diff. I've seen this happening before

Reply:
Replaced tabs with spaces. Please check now